### ISSUE TITLE
Automatic Comment on Pull Request Raised

### ISSUE THREAD

**Issue by thevijayshankersharma** [Niketkumardheeryan/ML-CaPsule#820]
Often, when pull requests (PRs) are raised, reviewers may lack crucial context about the changes proposed. This can lead to delays in the review process and misunderstandings between contributors and reviewers.

**Comment by Niketkumardheeryan** [comment]
@thevijayshankersharma please ensure that message must be According to Project guidelines.

**Comment by thevijayshankersharma** [comment]
@Niketkumardheeryan Yeah sure. Even though if there is an issue we will clarify it on my PR.
